### PR TITLE
Integer support

### PR DESCRIPTION
**Pull request by mohamedsayed18**
Adding support for Int32 data type

### CODE REVIEW

File: ros_ign_bridge/include/ros_ign_bridge/builtin_interfaces_factories.hpp [hunk 1]
```diff
@@ -38,6 +38,7 @@
 #include <std_msgs/msg/empty.hpp>
 #include <std_msgs/msg/float32.hpp>
 #include <std_msgs/msg/string.hpp>
+#include <std_msgs/msg/int32.hpp>
```
Comment (chapulina) [inline ros_ign_bridge/include/ros_ign_bridge/builtin_interfaces_factories.hpp:40]: Nitpick:keep alphabetic order.

The same goes for other files.

**Review by chapulina — COMMENTED** [review]
Works for me! I tried with:

* Terminal 1: `ros2 run ros_ign_bridge parameter_bridge /chatter@std_msgs/msg/Int32@ignition.msgs.Int32`

* Terminal 2: `ros2 topic echo /chatter`

* Terminal 3: `ign topic pub -t /chatter -m ignition.msgs.Int32 -p 'data:1'`

Just one other request, could you list the new message in the table in the [README](https://github.com/ignitionrobotics/ros_ign/blob/ros2/ros_ign_bridge/README.md#L9-L35)?

**Comment by mohamedsayed18** [comment]
I added Int32 to the readme and fixed the alphabetic order

**Review by chapulina — APPROVED** [review]
Thanks, LGTM!

**Pull request merged**